src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  border-radius: 4px;
+}
+
+.search-bar,
+.search-bar__field,
+.search-bar__input {
+  width: 100%;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
1 - you have other styles for this classes, so why not write these styles to them?
2 - are you sure that all of these elements have default paddings?
3 - in that case better to set `box-sizing` to `*` selector